### ISSUE TITLE
[Crash] Unable to log into paper server, through velocity, when installed on client. 

### ISSUE THREAD

**Issue by purejosh** [Luke100000/ImmersivePaintings#11]
Getting an invalid payload when trying to log in through velocity to a paper server. A proxy including Modded Fabric MC is really only ever supported with Velocity, so if there's a way to resolve this, that would be awesome. 

Let me know if I can provide anything for you. I'd be eager to see this working.

**Comment by Luke100000** [comment]
Take a look at https://github.com/Luke100000/ImmersivePaintings/wiki/Config
Does it work if you set `packetSize` to a very small number, let's say 1024 to make sure? For both server and client. If it works, can you please test a little bit on what's a safe number? I assume it should be slightly below 16 KB.

**Comment by purejosh** [comment]
Ah, lemme give that a try.

**Comment by purejosh** [comment]
Oh, I may have failed to explain what was happening correctly. I'm sorry about that. It's been a long 3-4 weeks of adding 1.19.2 mods to the Fabric server, I'm a mess. 

So, I actually don't get any errors when I log in to the proxy and it directs me to the Fabric Modded server, since that was the last server I was on. I actually get the error when I log in, and the last world I was logged into was the lobby, or survival world, which are both paper. 

So, tl;dr, the issue is there when logging into a paper server, through a velocity proxy, while this mod is installed on the client.
